tox.ini

```
@@ -3,6 +3,8 @@ isolated_build = true
 skip_missing_interpreters = True
 minversion = 3.25.1
 requires =
+    # Ensure tox and virtualenv compatible back through 2.7.
+    tox<4
```

To make clear you're not talking about Tox 2:

```suggestion
    # Ensure tox and virtualenv compatible back through Python 2.7.
    tox<4
```

---

Fixed.